Sources/TuistSupport/Logging/OSLogHandler.swift

```diff
@@ -1,5 +1,5 @@
 import Foundation
-import Logging
+import struct Logging.Logger
```

Comment:
nice, didn't realize defining it this way helps the compiler disambiguate 

Do we need to have 2 imports? one for `Logging` and another for the `struct Logging.Logger` explicitly? 

This file uses other types from the `Logging` framework - e.g. `LogHandler`

Reply:
Since you can't import individual types this _does_ import the whole Logging module, but it explicity denotes that the Logger type should come from this module

Reply:
TIL 👍